packages/react/src/components/Grid/Column.js

```diff
@@ -124,6 +125,13 @@ function getClassNameForBreakpoints(breakpoints) {
 
     const name = breakpointNames[i];
 
+    // If our breakpoint is a boolean, the user has specified that the column
+    // should be "auto" at this size
+    if (typeof breakpoint === 'boolean') {
+      classNames.push(`${prefix}--col-${name}`);
```

Comment:
last comment: there appears to be a minor difference between `${prefix}--col-${name}` and `${prefix}--col-${name}--auto` of respectively having `width: 100%` and `width: auto`. I presume we should be using the class with auto but if this change was intentional, I'm curious about why!

Reply:
Oh wild, let me take a look real quick to see why 👀 Thanks for the catch!

Reply:
So far couldn't really find anything 🤔 Will keep digging but will move on with the `width: 100%` unless there is a good reason to use the auto classes.